robocop/checkers/lengths.py

```diff
@@ -227,11 +227,22 @@
         """,
         severity=RuleSeverity.ERROR,
     ),
-    "0529": Rule(rule_id="0529", name="empty-test-template", msg="Test Template is empty", severity=RuleSeverity.ERROR),
+    "0529": Rule(
+        rule_id="0529",
+        name="empty-test-template",
+        msg="Test Template is empty",
+        docs="""
+        The ``Test Template`` set template to all tests in the suite. Empty value is considered an error
+        because it leads users to wrong impression on how the suite operates.
+        Without value, the setting is ignored and tests are not templated.
```

Comment:
```suggestion
        ``Test Template`` sets the template to all tests in a suite. Empty value is considered an error
        because it leads the users to wrong impression on how the suite operates.
        Without value, the setting is ignored and the tests are not templated.
```